### DISCUSSION TITLE
When using AutoComplete freeSolo Material-UI, press enter submit the form

### DISCUSSION THREAD

**Discussion by pramade** [react-hook-form/react-hook-form#2772] [Q&A]
Hello, 
I'm facing a problem when using Autocomplete component of Material-UI.
I'm using a freeSolo Autocomplete to be able to pass a string.
When selecting a value from the option or adding a value by clicking on it, it doesn't trigger the submit.
However when writing a string and press enter it trigger the submit and I would like to get ride of that.
Do you have any advice ?

There is the sandbox :
https://codesandbox.io/s/react-hook-form-usefieldarray-forked-f2wu0

Thank you :)

**Comment by Biz-e-Services** [comment]
Hi Pramade.
 I reverted to using as={props... instead of render={props... It seems to work but I have other issues on the validation side of things. Hopefully Bill or RHF team will help us with these issues.
